examples/seismic/viscoacoustic/operators.py

```diff
@@ -42,29 +42,30 @@ def src_rec(p, model, geometry, **kwargs):
     return src_term + rec_term
 
 
-def sls_1st_order(model, geometry, p, **kwargs):
+def sls(model, geometry, p, **kwargs):
```

Comment:
I preferred it split into `1st_order` and `2nd order` lot easier to read and find things, you can always make a time order map to know which one to use (see how tti switch kernels for example).

Reply:
We agree with you @mloubout when you say that is lot easier to read, our first implementation followed this approach. But following @georgebisbas suggestion, we believe that all code be more clean (without repetitions) and easy to understand, once order map is not necessary. We believe that both approaches are valid and if you don't agree we could undo this last modification.

Reply:
I think I agree with Mathias @speglich 

I think the only situation in which we should factorize these functions is when the maths allows you to do so. So for example if a certain operator is a generalisation of the other, then go fo it (eg OT2 - OT4 kernels in iso-acoustic). But otherwise, keeping it split improves readability, even if the price to pay is some redundant code (and this is just setup code, not algorithms, so it's easier to swallow...)